WaterNetworkManagement/Simulation/Simulation/README.md

```diff
@@ -0,0 +1,7 @@
+# Simulation
+
+## Description 
+
+This entity contains a harmonised description of a Simulation made for
```

Comment:
harmonised description of a simulation *scenario*? Could also change entity name to `SimulationScenario` to improve clarity. If doing this, also change the `hasSimulation` relationship in the Result entity to `hasSimulationScenario`, and potentially change the `Result` entity to `SimulationResult`.